components/mixins/experience-event/events/clickemail.schema.json

```diff
@@ -0,0 +1,123 @@
+  "title": "Click Email",
+  "type": "object",
+  "meta:conditionalField": "xdm:eventType",
+  "meta:conditionalValue": "directMarketing.clickEmail",
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/experienceevent"
+  ],
+  "description": "Use for information regarding direct marketing click email events.",
+  "definitions": {
+    "clickemail": {
+      "properties": {
+        "xdm:directMarketing": {
+          "title": "Direct marketing",
+          "type": "object",
+          "properties": {
+            "xdm:mailingID": {
+              "title": "Mailing ID",
+              "type": "integer",
+              "description": "Unique identifier of the mailing."
+            },
+            "xdm:mailingName": {
+              "title": "Mailing Name",
+              "type": "string",
+              "description": "Friendly name for the mailing."
+            },
+            "xdm:linkURL": {
+              "title": "Link URL",
+              "type": "string",
+              "description": "The link URL."
+            },
+            "xdm:testVariantID": {
+              "title": "Test Variant ID",
+              "type": "integer",
+              "description": "Unique identifier for the test variant."
+            },
+            "xdm:testVariantName": {
+              "title": "Test Variant Name",
+              "type": "string",
+              "description": "Friendly name for the test variant."
+            }
+          }
+        },
+        "xdm:environment": {
+          "title": "Environment",
+          "type": "object",
+          "description": "The information related to web page and link of the ExperienceEvent.",
+          "properties": {
+            "xdm:operatingSystem": {
+              "title": "Operating system",
+              "type": "string",
+              "description": "The name of the operating system used when the observation was made. The attribute should not contain any version information such as '10.5.3', but instead contain 'edition' designations such as 'Ultimate' or 'Professional'."
+            },
+            "xdm:browserDetails": {
+              "title": "Browser details",
+              "type": "object",
+              "description": "The browser specific details such as browser name, version, javascript version, user agent string, and accept language.",
+              "properties": {
+                "xdm:userAgent": {
+                  "title": "User agent",
+                  "type": "string",
+                  "description": "The HTTP user-agent string from the client request."
+                }
+              }
+            }
+          }
+        },
+        "xdm:device": {
+          "title":"Device",
+          "type": "object",
+          "properties": {
+            "xdm:isMobileDevice": {
+              "title": "Is Mobile Device",
+              "type": "boolean",
+              "description": "Is this mobile device."
+            },
+            "xdm:model": {
```

Comment:
Is this xdm:model or xdm:device (on the wiki)?

Reply:
xdm:model is the existing property in Device data type and so i am mapping to xdm:model . As we discussed in our meeting please let me know once you have more information